api/app/Http/Helper/Mail/helpers.php

```diff
@@ -0,0 +1,46 @@
+<?php
+ /*this is a common method for sending mails to users
+ data : users data
+ subject : whatyour mail subject should be
+ to_list : whom is the mail for
+ cc_list : list of people in cc list (optional)
+ bcc_list : list of people in bcc list (optional)
+ */
+
+function send_mails($data,$subject,$to_list,$cc_list=NULL,$bcc_list=NULL)
+{
+	
+	if (!empty($cc_list) && !empty($bcc_list)) 
+	 {
+	 	//if bcc cc list is given
+	 	Mail::send( $data['redirect_url'], ['violation_data' => $data], function($message) use($to_list,$cc_list,$bcc_list,$subject){
```

Comment:
@jalmiShreya - violation_data pr redorect_url as an attribute to general send_mail won't make sense.

Can we have general names such as "template_name" and "template_data" instead?